conda/environments/cuml_dev_cuda10.0.yml

```diff
@@ -0,0 +1,21 @@
+name: cuml_dev
+channels:
+- rapidsai
+- nvidia
+- rapidsai-nightly
+- conda-forge
+dependencies:
+- cudatoolkit=10.0
+- libclang=8.0.0
+- cmake=3.14.5
+- numba=0.44*
+# - cupy>=6*
+- cudf=0.8*
+- rmm=0.8*
+- cython>=0.29,<0.30
+- pytest>=4.6
+- scikit-learn=0.21
+- umap-learn>=0.3.9
+- scikit-learn>=0.21
+- dask
+- dask-cuda=0.8*
```

Comment:
Can we add jupyter notebook here as well?

Reply:
Not sure if its a good idea, the idea is the minimal packages required for cuml. You can install jupyter in the base of conda and then use `nb_conda` package to be able to select different kernels that are different environments which is a much better solution than installing jupyter in the environments

Reply:
Don't think most of the people would know what you described here. I'd add the jupyter notebook which we need pretty much all the time.

Reply:
We don't need it for development all the time and can conflict with other types of workflows like the one I described (that is not that uncommon really)

Reply:
Trying to keep the conda file useful but without many non essential files, jupyter is useful but not essential and very easy to install on top

Reply:
What we could do is a follow up PR where we create a more full featured conda file that includes jupyter, perhaps things like matplotlib and seaborn and other useful things in something called `cuml-workflow` yml file so that we have a "lean" conda recipe here, and a full featured convenience one as well

Reply:
Another yml file works for me.